build/x86_64_centos6/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM centos:6
+
+RUN yum update -y && yum install -y \
+  git \
+  curl \
+  which \
+  procps \
+  gcc-c++ \
+  zlib-devel \
+  redhat-lsb-core \
+  glibc-devel \
+  patch \
+  bzip2 \
+  rpm-build \
+  rpm-sign \
+  autoconf \
+  expect \
+  centos-release-SCL
+
+ENV PATH ${PATH}:/usr/local/bin
+
+RUN rpm -ivh http://repoforge.mirror.constant.com/redhat/el6/en/x86_64/rpmforge/RPMS/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm \
```

Comment:
These files seem to have a lot more layers than the Debian one. Any reason we can't fold all of these RUN commands into one?

Reply:
Combined them. PTAL